### DISCUSSION TITLE
Can we set a $native or $web in config?

### DISCUSSION THREAD

**Discussion by bmccabe96** [tamagui/tamagui#1841] [Q&A]
Hello, my team at work is using tamagui to write shared components across web and mobile. We use expo for our native app and next js for our web app. 

I’m fairly new to tamagui and have been playing around with it. My question is: similar to how we can set styling options for $sm, $md etc based on screen width, is there any way to set something that detects native vs web? It would be cool to do a $native as opposed to having conditionals within the tsx that do something like Platform.OS !== “web” etc etc.

**Comment by nderscore** [comment]
https://tamagui.dev/docs/intro/props#platform

**Reply by octalpixel** [reply]
Weird facing and issue where it works locally but when putting a production nextjs, $platform styles do not get applied.
